### ISSUE TITLE
ti_misp - Integration making constant API calls to MISP instance

### ISSUE THREAD

**Issue by ChestoOfGlen** [elastic/integrations#6510]
Hi all,

I have noticed since the recent update to v1.15.1 for the MISP integration, that it is making constant requests to our MISP API even though it is returning no data. We are currently running it on Elastic agent 8.7.1 in Kubernetes (via ECK).
This is causing a flow-on effect that is severely affecting our MISP instance.
![image](https://github.com/elastic/integrations/assets/37160206/21fa1cdf-3ec6-4df0-8408-c7e6a7a838e4)

We applied the latest package and policy updates automatically just after 9AM GMT+10 yesterday (Wednesday) and almost immediately our MISP cluster got pegged at 100% CPU utilisation and very high CPU load.
![image](https://github.com/elastic/integrations/assets/37160206/b827705e-dcf8-4093-81dc-125bc130a100)

I have tried both the default MISP API and the Attributes API with the same results.
I have currently tried to test using one instance and a very short initial lookback time of "1h", but even this is still resulting in seeing a constant stream of 8-10 requests per second.
![image](https://github.com/elastic/integrations/assets/37160206/7f7558ff-6143-4777-b0c8-a8276fb5d38c)

I temporarily enabled request tracing to further review the requests/response with out 1 hour lookback.
From what I see in the tracing logs, the integration is looping through a (seemingly) random number of pages even though it is returning zero results for all of them.
These are the first request/response I observed in the logs.
![image](https://github.com/elastic/integrations/assets/37160206/de41912c-47ab-4a50-a5a7-67f88bb70c2c)
![image](https://github.com/elastic/integrations/assets/37160206/9e631b34-52f7-427c-b8e0-9e0336ab7f07)

These are the last request/response I observed before turning off the tracing.
![image](https://github.com/elastic/integrations/assets/37160206/d1bec816-730f-479e-938f-5aa1d710ba3f)
![image](https://github.com/elastic/integrations/assets/37160206/1dc7327e-29af-4ce5-a070-2c5af7d68cd5)

In short, I don't know what exactly has changed, or why it is now trying to loop through a (seemingly) random number of pages of results that all return no attributes.

The v1.15.1 pull request (https://github.com/elastic/integrations/pull/6482) talks about changing some timestamps, but I fail to see how that would have this effect on the integration.

Any assistance would be greatly appreciated.

**Comment by andrewkroh** [comment]
This does not look like a new problem caused by #6482, but perhaps the upgrade triggered a restart of the integration.

This is an example of an empty response from the attributes/restSearch API.

```
% curl -k -XPOST -H "Authorization: hM1it4KA56pLDW5jJmlXTKZMsFV9RBpeSj3ZCu1Z" -H "Content-Type: application/json" "https://localhost/attributes/restSearch" -d '{"limit": 10, "page":33000}'
{"response": {"Attribute": []}}
```

But both the httpjson config and the mock server have it wrong. So instead of breaking the pagination loop it continues.

https://github.com/elastic/integrations/blob/e264e76bba4d73e1aabe3ee913864691584d28b3/packages/ti_misp/data_stream/threat_attributes/agent/stream/httpjson.yml.hbs#L51

https://github.com/elastic/integrations/blob/f38ba1adb5d34c8de64974b22647cfe28ea1bdcc/packages/ti_misp/_dev/deploy/docker/files/config.yml#L475-L478
